e2e/__tests__/__snapshots__/testFailing.test.ts.snap

```diff
@@ -41,14 +41,12 @@ exports[`works with all statuses 1`] = `
       28 | });
       29 |
     > 30 | test.failing.each([
-         | ^
+         |              ^
```

Comment:
this change is perfectly fine. would be even better to point at `failing`, but as long as it points to the declaration and not below the data I'm happy 😀

Reply:
This is difficult unless we completely control `source-map-support` ourselves.😀

Reply:
makes sense. And our usage of it is apparently broken anyways (see #9147)